tests/e2e/machine_pool_test.go

```diff
@@ -353,6 +353,78 @@ var _ = Describe("TF Test", func() {
 
 			})
 		})
+		Context("Author:amalykhi-High-OCP-65063 @OCP-65063 @amalykhi", func() {
+			It("Author:amalykhi-High-OCP-65063 Create single-az machinepool for multi-az cluster", ci.Day2, ci.High, ci.FeatureMachinepool, func() {
+				if !profile.MultiAZ {
+					Skip("The test is configured for MultiAZ cluster only")
+				}
+				getResp, err := cms.RetrieveClusterDetail(ci.RHCSConnection, clusterID)
+				Expect(err).ToNot(HaveOccurred())
+				azs := getResp.Body().Nodes().AvailabilityZones()
+				By("Create additional machinepool with availability zone specified")
+				replicas := 3
+				machineType := "r5.xlarge"
```

Comment:
Better to list the machineTypes from AWS and prepare it from the list result. There is aws_inquiries endpoint in OCM.

Reply:
@xueli181114 I got it. May I refactor all the machinepool tests with this enhancement in the next PR?

Reply:
@amalykhi Of course. Then I will LGTM it.